spec/models/project_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe Project, type: :model do
+  context "invalid params" do
+    it "does not save project without title" do
+      project = Project.new(date: "20-07-2016", description: "o niczym")
+      expect(project.save).to eq(false)
+    end
+
+    it "does not save project with too long title" do
+      project = Project.new(title: "projekt o bardzo długim, długim, długim, długim, długim, długim, długim, długim za długim tytule o kilka znaków", date: "20-07-2016", description: "o niczym")
```

Comment:
when you want to create long string you can write
example_string = "long string" * 100